src/sentry/models/integration.py

```diff
@@ -62,6 +63,35 @@ class Meta:
         unique_together = (("project", "stack_root"),)
 
 
+class ExternalProviders(Enum):
+    GITHUB = 0
+    GITLAB = 1
+
+
+EXTERNAL_PROVIDERS = {
+    ExternalProviders.GITHUB: "github",
+    ExternalProviders.GITLAB: "gitlab",
+}
+
+
+class ExternalTeam(DefaultFieldsModel):
+    __core__ = False
+
+    team = FlexibleForeignKey("sentry.Team")
+    provider = BoundedPositiveIntegerField(
```

Comment:
Every model in `src/sentry/models/` that has a `provider` column is a `models.CharField`. Should they migrate to enums as well or are we doing something different with this `provider`?

Reply:
@mgaeta I'm not sure why they are setup as varchar types. Enum is the ideal data structure for this field because it only be one of those 2 values. And the other `provider` fields probably should be enums; I don't have the historical context for why decision was made for `varchar`.